### PR TITLE
Make FreeOrionNode global so it can be used in other scenes

### PR DESCRIPTION
**Pull request by o01eg**

### CODE REVIEW

File: client/godot/FreeOrionNode.cpp [hunk 1]
```diff
@@ -391,3 +401,8 @@ godot::Dictionary FreeOrionNode::get_fleets() const {
     return fleets;
 }
 
+void FreeOrionNode::send_chat_message(godot::String text) {
+    std::string text8 = std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>, wchar_t>{}.to_bytes(text.unicode_str());
```
Comment (geoffthemedio) [inline client/godot/FreeOrionNode.cpp]: Does Godot use UTF-16 internally on all platforms?

**Comment by geoffthemedio** [comment]
Otherwise no comment

File: client/godot/FreeOrionNode.cpp (hunk 1 above)
Comment (o01eg) [reply client/godot/FreeOrionNode.cpp]: No, it uses `wchar_t` but `std::codecvt_utf8_utf16` looks like works both with Linux 4-bytes and Windows UTF-16 representation.

Comment (o01eg) [reply client/godot/FreeOrionNode.cpp]: Refactored it.

**Pull request merged**